dbt/models/qc/qc.vw_iasworld_sales_latest_sale.sql

```diff
@@ -0,0 +1,21 @@
+SELECT
+    parid,
+    instruno,
+    price,
+    saledt,
+    DATE_FORMAT(
+        DATE_PARSE(saledt, '%Y-%m-%d %H:%i:%S.%f'), '%c/%e/%Y'
+    ) AS saledt_fmt
```

Comment:
A number of different QC reports reformat iasworld dates in a similar fashion. I thought about factoring out the logic into a dbt macro, or trying to figure out how to get Excel to recognize the strings as dates in order to format them natively, but this ended up feeling to me like the least complex approach.